go/client/cmd_passphrase_recover.go

```diff
@@ -118,8 +89,8 @@ func (c *CmdPassphraseRecover) GetUsage() libkb.Usage {
 	}
 }
 
-func (c *CmdPassphraseRecover) errNoUID() error {
-	return errors.New(`Can't recover without a UID.
+func (c *CmdPassphraseRecover) errNotProvisioned() error {
```

Comment:
are these errors still used/needed?

Reply:
Readded them, legacy flow didn't change much so it should stay as-is.

Reply:
can you add a TODO to move these to UI calls when `legacyProvisioning` is removed/refactored for the GUI?

Reply:
I'll wait with this for a meeting so that we have a consensus on how we should structure this. IIRC we want to work on this in core, so it might make sense to bundle it up with the frontend impl.

Reply:
right i'm just saying to mark it with a ticket so we can remember to clean it up.